### ISSUE TITLE
Integer minimal is ignored

### ISSUE THREAD

**Issue by MrGeneration** [zammad/zammad#2339]
### Infos:

* Used Zammad version: 2.7
* Installation method (source, package, ..): n/a
* Operating system: n/a
* Database + version: n/a
* Elasticsearch version: n/a
* Browser + version: verified on Google Chrome and Firefox, most likely on all Browsers

### Expected behavior:

* When setting a minimal value to a integer object (something like 6 or 1000), Zammad will throw an error and not update the ticket, if you enter a smaller integer like e.g. 2.

### Actual behavior:

* When setting a minimal value to a integer object (something like 6 or 1000), Zammad will simply update the ticket without any hassle, even when entering a smaller integer like e.g. 2.

### Steps to reproduce the behavior:

* Create a new object with integer and a minimal value higher than 1 (Object type does not matter)
* Update the database and reload the webapp
* Open ANY ticket (or user, organization depending on what you chose) and try to update the integer object with a value smaller than the minimal value you defined.

This impacts Ticket#1029915

Yes I'm sure this is a bug and no feature request or a general question.

**Comment by thorsteneckel** [comment]
Hi @SummerStorm - is this a duplicate and already covered by our internal ObjectManager::Attribute refactoring branch?

**Comment by SummerStorm** [comment]
@thorsteneckel No, it is not a duplicate. Our internal ObjectManager::Attribute refactoring branch only covers required attributes and data/datatime attributes so far. I decided to keep the ObjectManager::Attribute refactoring branch lean and simple so that other attribute types can be added in one-by-one after the main attribute validation framework is in place. This way the validation patch for each individual attribute type is its own commit on `develop` so it's easier to fix/modify/upgrade down the road.

I'll add the validation for this once the main ObjectManager::Attribute refactoring branch has been merged.
